src/design-tokens/tier-2-usage/colors-border.json

```diff
@@ -70,7 +70,12 @@
               "value": "{eds.color.other.mint.300}"
             },
             "revise": {
-              "value": "{eds.color.other.orange.100}"
+              "default": {
```

Comment:
should these be flipped so `subtle` is lighter than the `default`?

Reply:
Yea, that makes sense, will flip